src/_product-updates/2024-02-19-new-dashboard-experience.md

```diff
@@ -0,0 +1,21 @@
+---
+title: New dashboard experience
+tags: [Improvement]
+image: /images/changelog/new-dashboard-2024.jpg
+---
+
+![new-dashboard-2024.jpg](/images/changelog/new-dashboard-2024.jpg)
+
+It's been a big project that we've poured a lot of energy and even more love into, and our new dashboard is finally available to everyone.
+It's become a true home for your API ecosystem and everything that happens within it. From creating a hub to exploring your documentation, everything has been redesigned to offer you the best possible experience.
+
+We've also put the finishing touches on another major change: the removal of individual spaces. To discuss this, let's take a step back.
+
+In its early days, Bump.sh was designed for individual developers but quickly became collaborative. The vast majority of our users now work in teams.
+Until now, each user had their own personal space and could create and/or join organizations. However, in practice, there was often confusion between the two, and the experience wasn't the most comfortable.
```

Comment:
Maybe we could say that "in practice, most of the personal workspaces were remaining empty, and there was often confusion between..."